tests/jeos/firstrun.pm

```diff
@@ -44,6 +49,24 @@ sub verify_user_info {
     assert_script_run("ldd --help | grep '^" . $lang_data{$proglang} . "'");
 }
 
+sub verify_mounts {
+    my $expected = {mount_type => (expect_mount_by_uuid) ? 'UUID' : 'LABEL'};
+
+    my @findmnt_entries = grep { /\s$expected->{mount_type}.*\stranslated\sto\s/ }
+      split(/\n/, script_output('findmnt --verbose --verify'));
+
+    diag "\nfindmnt ulitity found ( " . scalar(@findmnt_entries) . " ) entries\n";
+    (scalar(@findmnt_entries) > 0) or die "Expected mounts by $expected->{mount_type} has not been found\n";
+
+    @findmnt_entries = grep { !/^$expected->{mount_type}/ }
```

Comment:
Why `--df`?

Reply:
I like the output format, mainly the lines start with mount type which is easy to process afterwards.